### ISSUE TITLE
UPDATE for 1.8

### ISSUE THREAD

**Issue by advocaite** [Fenixin/Minecraft-Region-Fixer#35]
please Update nbt data fro 1.8

**Comment by sigod** [comment]
Doesn't work with world from 1.8 server version. It says this:
```
d:\sigod\Minecraft-Region-Fixer-0.1.3>region-fixer.exe d:\sigod\world

Welcome to Region Fixer!
(version: 0.1.3)

############################################################
#################  Scanning world: world  ##################
############################################################

Scanning directory...
There are 9 region files in the world directory.

-------------------- Checking level.dat --------------------
'level.dat' is readable

------------------ Checking player files -------------------
Info: No player files to scan.

------------------ Scanning the Overworld ------------------
Scanning:  9 /  9 100% [#######################################] Time: 00:00:03

Chunk problems:
-------------------------------------------------------------------
| Problem | Corrupted  Wrong l.  Etities  Shared o.  Total chunks |
-------------------------------------------------------------------
| Counts  |     0         4         0        103         3400     |
-------------------------------------------------------------------

Region problems:
No problems found.
```
When there's ~20 corrupted chunks.

**Comment by aetaric** [comment]
This is confirmed here as well. I have run the latest branch against my world and it says all is well. On the next start up of the server I am flooded with chunks in wrong location errors.

**Comment by Fenixin** [comment]
Hello!

@sigod, could you tell me how you know there are 20 chunks corrupted? When you mean corrupted is [https://github.com/Fenixin/Minecraft-Region-Fixer/wiki/Chunk-and-region-problems#corrupted](corrupted) or "oh my god this doesn't work"? Could you also try the beta-2-2 and see if that gets the expected results?

@aetaric, could you also try beta-2-2?

**Comment by JuliB15** [comment]
Dead urls... Where in the whole internet I can found a beta version?

**Comment by Fenixin** [comment]
Here:

http://www.minecraftforum.net/forums/support/server-support/server-administration/1903200-minecraft-region-fixer

But is no longer in beta... also, I should close this!

**Comment by JuliB15** [comment]
Beta-2-2 is Version 0.2.0? Version 0.2.0 ignores the UUID files of 1.8+!

**Comment by sigod** [comment]
@JuliB15 What do you mean by UUID files?

**Comment by JuliB15** [comment]
I'm sorry, I would say the programm doesn't scan the playerdata, but they are named with uuid.dat (0hbqwd90-1289e.dat or like this). It says "No files to scan". There are exact 997 files to scan! Am I wrong?

**Comment by Fenixin** [comment]
@JuliB15, regionfixer is no longer in beta, version 0.2.0 is the last version of the program and it should scan UUID files. Is your world folder modded in any way? If you assure me that your world is not modded, I will give it a look as soon as I get some time.

**Comment by JuliB15** [comment]
I like sympathetic programmers!
Given path is   server/world
Are parameters important?
Version is 0.2.0, the tree looks like

-Server
   +world
      *playerdata
         -12345xyz.dat
         -etc.dat
      *region
      *level.dat
      *etc
   +world_nether
   +world_etc

output:
Playerdata
"No files to scan"

**Comment by Fenixin** [comment]
@JuliB15, You are right, there is some issue and it skips UUID players. I will look into it soon.

Thanks for reporting!

**Comment by JuliB15** [comment]
That is nice! Thx!

**Comment by Fenixin** [comment]
Hello!

@JuliB15, Sorry for the time a took to do this. Your edit with the pressure went unnoticed as github didn't send me an email. The problem was really simple and its fixed now in master (v0.2.1).

**Comment by JuliB15** [comment]
No Problem!!! You rescued my server! THANK YOU!
